packages/protocol/tasks/deploy_L1.ts

```diff
@@ -184,6 +184,18 @@ export async function deployContracts(hre: any) {
         );
     }
 
+    // SignalService
+    const SignalService = await deploySignalSerive(hre, AddressManager.address);
+
+    // Used by Bridge
+    await utils.waitTx(
+        hre,
+        await AddressManager.setAddress(
+            `${chainId}.signal_service`,
+            Bridge.address
```

Comment:
`Bridge.address` is not correct, we should deploy a SignalService then use its address.

Reply:
fixed